packages/buidler-core/src/internal/core/runtime-environment.ts

```diff
@@ -135,11 +143,14 @@ export class Environment implements BuidlerRuntimeEnvironment {
     let runSuper: RunSuperFunction<TaskArguments>;
 
     if (taskDefinition instanceof OverriddenTaskDefinition) {
-      runSuper = async (_taskArguments = taskArguments) =>
-        this._runTaskDefinition(
+      runSuper = async (_taskArguments = taskArguments) => {
+        log("Running %s's super", taskDefinition.name);
```

Comment:
Can we get the super's name here?

Reply:
They have the same name. We should probably store info about which plugin defined each task.